module/User/src/User/Service/ApiApp.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace User\Service;
+
+use Application\Service\AbstractService;
+use Firebase\JWT\JWT;
+use User\Mapper\ApiApp as ApiAppMapper;
+use User\Model\User as UserModel;
+
+class ApiApp extends AbstractService
+{
+
+    /**
+     * @var ApiAppMapper
+     */
+    protected $mapper;
+
+    /**
+     * Constructor
+     *
+     * @param ApiAppMapper $mapper
+     */
+    public function __construct(ApiAppMapper $mapper)
+    {
+        $this->mapper = $mapper;
+    }
+
+    /**
+     * Get a callback from an appId and a user identity
+     * @param string $appId
+     * @param UserModel $user
+     * @return string
+     */
+    public function callbackWithToken($appId, UserModel $user)
+    {
+        $app = $this->getMapper()->findByAppId($appId);
+
+        $token = [
+            'iss' => 'https://gewis.nl/',
```

Comment:
The `iss` field is the issuer field. As described in [RFC 7519](https://tools.ietf.org/html/rfc7519#section-4.1.1).

It doesn't need to be verified. And will not be verified (thus, it can be used on `test.gewis.nl`). It is just to show that the token was issued by the GEWIS Website.